xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-index/src/main/java/org/xwiki/extension/index/internal/security/SecurityIssueDescriptor.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.extension.index.internal.security;
+
+import org.xwiki.extension.version.Version;
+
+import us.springett.cvss.Cvss;
+
+/**
+ * An individual security issue descriptor.
+ *
+ * @version $Id$
+ * @since 15.5RC1
+ */
+public class SecurityIssueDescriptor
```

Comment:
Wouldn't "vulnerability" be a bit more accurate than "issue" ? "issue" tend to relate to jira issues which is not what this is about AFAIU.